CMakeLists.txt

```diff
@@ -598,6 +598,15 @@ endif()
 target_link_libraries(tvm PRIVATE ${TVM_LINKER_LIBS} ${TVM_RUNTIME_LINKER_LIBS})
 target_link_libraries(tvm_runtime PRIVATE ${TVM_RUNTIME_LINKER_LIBS})
 
+if(BUILD_FOR_HEXAGON AND DEFINED USE_HEXAGON_GTEST AND EXISTS ${USE_HEXAGON_GTEST})
```

Comment:
`USE_HEXAGON_GTEST` should likely be a [tvm_option](https://github.com/apache/tvm/blob/fa19d826f1c286e2c9ccc8770ff00cd71ddf49a1/CMakeLists.txt#L45) and should appear in `cmake/config.cmake` given that it effects the build,

Reply:
Done.